### PR TITLE
Bug 1419957 - Remove old style backfill/add new job features

### PR DESCRIPTION
**Pull request by edmorley**
Since everything but standard retrigger/cancellation is now handled client-side by tcactions, making the backend parts that provided pulse messages to pulse_actions redundant (since it was decommissioned in bug 1379172).

### CODE REVIEW

**Comment by edmorley** [comment]
(Ignore the selenium test failure, it's bug 1421111)

File: ui/plugins/controller.js [hunk 1]
```diff
@@ -381,6 +380,7 @@ treeherder.controller('PluginCtrl', [
                 return;
             }
 
+            // TODO: Figure out if this conditional is needed any more
```
Comment (imbstack) [inline ui/plugins/controller.js]: Perhaps the thing to do with it now is pop up an alert saying something like "no system exists to handle this action for this job type" if the conditional is false?

**Comment by edmorley** [comment]
> Have you had a chance to try it locally yet?

To be honest I've never really used most of these features, so I don't really know what to expect when testing. Perhaps deploying to stage might be the best way forwards (does mozilla-taskcluster still listen there, probably not?).

**Comment by imbstack** [comment]
Testing on stage seems like a good plan. I just checked and mozilla-taskcluster-staging seems to listen on `exchange/treeherder-stage/v1/job-actions`, so I think we're ok to test those actions there?

File: ui/plugins/controller.js (hunk 1 above)
Comment (edmorley) [reply ui/plugins/controller.js]: Good idea - I've added a:
`thNotify.send('Unable to backfill this job type!', 'danger', { sticky: true });`

**Comment by edmorley** [comment]
Brian kindly tested this on stage and found that retriggering new jobs on a try push gave a success message of just "e". I've updated the PR with what should fix that and can't repro on stage - does it now look ok to you @imbstack ? :-)

**Comment by imbstack** [comment]
Yep! Looks good to me now. r+

**Comment by edmorley** [comment]
Many thanks :-)

**Pull request merged**